### PR TITLE
CLI - Set default server to `testnet`

### PR DESCRIPTION
**Pull request by dylanh724**
# Description of Changes

`testnet` is quite stable now and makes onboarding (both the UX and docs) a significantly more-streamlined experience

- from "local"
- Suggested by @cloutiertyler 

image

# API and ABI breaking changes

(This is technically a breaking change? I'm not quite sure what label to apply)

# Expected complexity level and risk

*How complicated do you think these changes are? Grade on a scale from 1 to 5,
2

⚠️ There may be some tooling that assumes local if they never explicitly set it to testnet. 

# Testing

*Describe any testing you've done, and any testing you'd like your reviewers to do,
so that you're confident that all the changes work as expected!*

- [ ] *Try it in a new user env*

### CODE REVIEW

**Comment by dylanh724** [comment]
Specific poke to @jdetter - heard you're working on CLI stuff now!👋

File: crates/cli/src/config.rs [hunk 1]
```diff
@@ -107,7 +107,7 @@ pub struct RawConfig {
 
 const DEFAULT_HOST: &str = "127.0.0.1:3000";
 const DEFAULT_PROTOCOL: &str = "http";
-const DEFAULT_HOST_NICKNAME: &str = "local";
+const DEFAULT_HOST_NICKNAME: &str = "testnet";
```
Comment (jdetter) [inline crates/cli/src/config.rs:110]: We would need to update all 3 of these:
```
const DEFAULT_HOST: &str = "testnet.spacetimedb.com";
const DEFAULT_PROTOCOL: &str = "https";
const DEFAULT_HOST_NICKNAME: &str = "testnet";
```

**Review by jdetter — CHANGES_REQUESTED** [review]

**Comment by jdetter** [comment]
We originally had the CLI pointing to testnet by default, I don't remember the exact reason why we changed it back to local. Changing it back to testnet has some implications but we could possibly get this into the next release. I'll talk to Tyler about it today.

File: crates/cli/src/config.rs (hunk 1 above)
Comment (dylanh724) [reply crates/cli/src/config.rs:110]: image
Thanks, got it

**Comment by cloutiertyler** [comment]
> We originally had the CLI pointing to testnet by default, I don't remember the exact reason why we changed it back to local. Changing it back to testnet has some implications but we could possibly get this into the next release. I'll talk to Tyler about it today.

The reason was that testnet was not stable and we were afraid that people would be trying to do the tutorial while Testnet was down. Now that it has been more stable, I think this change makes sense.

**Review by jdetter — APPROVED** [review]
LGTM

**Review by jdetter — CHANGES_REQUESTED** [review]
I have to double check that the smoketests are working properly after this change

**Comment by jdetter** [comment]
We'll have to rework this PR after https://github.com/clockworklabs/SpacetimeDB/pull/1151 lands

**Review by jdetter — APPROVED** [review]
LGTM 👍

**Pull request merged**